qa/tasks/cephfs/test_mirroring.py

```diff
@@ -421,6 +421,34 @@ def test_directory_commands(self):
         self.disable_mirroring(self.primary_fs_name, self.primary_fs_id)
         self.mount_a.run_shell(["rmdir", "d1"])
 
+    def test_directory_command_ls(self):
```

Comment:
Test fails here: https://pulpito.ceph.com/vshankar-2024-08-20_10:11:38-fs-wip-vshankar-testing-20240820.042758-debug-testing-default-smithi/7863046/

Reply:
Fixed the bug and the test passed:
https://pulpito.ceph.com/jcollin-2024-08-29_23:33:17-fs:mirror-wip-jcollin-testing-distro-default-smithi/
https://pulpito.ceph.com/jcollin-2024-08-29_23:32:51-fs:valgrind-wip-jcollin-testing-distro-default-smithi/